public/javascripts/usage_bar_chart.js

```diff
@@ -2,22 +2,24 @@
 
     /*global d3, startDate, endDate, startTime, endTime, formatWeek, formatHour, numberToHumanSize, formatFixed, formatDate, formatTime, numberWithDelimiter */
 
-    function bar2Viz(data, div, count, format, displayMode, yop) {
+    function barWidth(width,length){
+      let calc_width = (width/(length)) - 2;
+      let bar_width = calc_width;
+      if (calc_width > 17){
+        bar_width = 17;
```

Comment:
@kjgarza in this implementation charts don't always have the same width. I strongly suggest to do that for consistency, using the minimal width that makes sense.

Reply:
I can make totally fix. It just looks odd (too thin, for no munch data). And I consider that there is a good range between 7-17px in which the bars look aesthetically pleasing.

Reply:
Is it too much work to show screenshots for the various bar widths so that we can decide as a team. As this has big implications for future work.

Reply:
Here are 2 examples


### ~7px (minimum)

https://search.test.datacite.org/works/10.7272/544d-1883

![image](https://user-images.githubusercontent.com/1092861/54996963-f68dff80-4fca-11e9-98c6-9a1f22e5071b.png)

### ~17px (Max)


https://search.test.datacite.org/works/10.7272/q6g15xs4

![image](https://user-images.githubusercontent.com/1092861/54997000-0ad1fc80-4fcb-11e9-8441-6f94a695e80c.png)

Reply:
we made all 7px 

fix 89baad49981f685e9a80612ac5b3065b5e7539c5